src/doghouse/core/ports/github_port.py

```
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Any, List, Optional
```

Ruff (0.15.7)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)